### DISCUSSION TITLE
EL3 Feedback

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#219] [Show and tell]
https://teambasedinquirylearning.github.io/precalculus/2024/EL3.html

Author: @kathypinzon 
Reviewer: @AbbyANoble

**Comment by kathypinzon** [comment]
@AbbyANoble I submitted this pull request this morning - sorry it was so late!

**Comment by AbbyANoble** [comment]
[Activity 5.3.1(b)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL3.html#act-define-log-3) - I might reword this a little. Maybe "Do you think P will have an inverse function? Why or why not?"  Or "P(t) has an inverse function. What characteristics seem to indicate that it does?"

**Comment by AbbyANoble** [comment]
[Activity 5.3.1(c)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL3.html#act-define-log-4) - Space needed between which and we in second to last sentence.

**Reply by AbbyANoble** [reply]
Also in this one, italicizing the word power may be helpful to lead them in the right direction.

**Comment by AbbyANoble** [comment]
[Definition 5.3.3](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL3.html#def-log) - Extra quotation mark right before "the logarithm..."

**Comment by AbbyANoble** [comment]
[Activity 5.3.7(a)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL3.html#act-find-log-2) - Typo in evaluate. And I don't think a comma is needed after logarithm in "Consider the logarithm log_3 9"

**Comment by AbbyANoble** [comment]
[Remark 5.3.10](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL3.html#EL3-3-12) - Replace the "A" with "The" in both paragraphs defining common and natural logs.
